javaslang/src/test/java/javaslang/collection/IntMap.java

```diff
@@ -23,7 +23,7 @@
     @SuppressWarnings("unchecked")
     public static <T> IntMap<T> of(Map<Integer, T> original) {
         return original.isEmpty() ? (IntMap<T>) EMPTY
-                                  : new IntMap<>(original);
+                : new IntMap<>(original);
```

Comment:
We provide a formatter for IntelliJ, see `.ide/idea-settings.jar`. Do you use it?